site/_includes/partials/devtools/es/whats-new.md

```diff
@@ -2,6 +2,21 @@
 
 Por favor, revisa la versión en inglés de <a href="/tags/new-in-devtools/" translate="no">What's New In DevTools</a> para una lista completa de las características lanzadas. Debajo tienes más contenido que ha sido traducido al castellano.
 
+### Chrome 95 {: #chrome95 }
```

Comment:
This seems unrelated but I guess was just missing.

Reply:
fixed